src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -4861,6 +4872,35 @@ namespace Akka.Util
         public Akka.Actor.Envelope Peek() { }
         public override string ToString() { }
     }
+    public sealed class MessageBuffer : System.Collections.Generic.IEnumerable<System.ValueTuple<object, Akka.Actor.IActorRef>>, System.Collections.Generic.IReadOnlyCollection<System.ValueTuple<object, Akka.Actor.IActorRef>>, System.Collections.IEnumerable
+    {
+        public int Count { get; }
+        public System.ValueTuple<object, Akka.Actor.IActorRef> Head { get; }
+        public bool IsEmpty { get; }
+        public bool NonEmpty { get; }
+        public Akka.Util.MessageBuffer Append(object message, Akka.Actor.IActorRef @ref) { }
+        public void DropHead() { }
+        public static Akka.Util.MessageBuffer Empty() { }
+        [return: System.Runtime.CompilerServices.TupleElementNamesAttribute(new string[] {
+                "Message",
+                "Ref"})]
+        public System.Collections.Generic.IEnumerator<System.ValueTuple<object, Akka.Actor.IActorRef>> GetEnumerator() { }
+    }
+    public sealed class MessageBufferMap<TId> : System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<TId, Akka.Util.MessageBuffer>>, System.Collections.Generic.IReadOnlyCollection<System.Collections.Generic.KeyValuePair<TId, Akka.Util.MessageBuffer>>, System.Collections.IEnumerable
```

Comment:
Any reason why MessageBufferMap should be exposed as a public API?

Reply:
changed to internal